### PR TITLE
Complete Mobile menu

### PR DESCRIPTION
**Pull request by Ibnballo1**
This is a completely responsive mobile menu.  The following were added to build the menu;

- I added a javascript file where I stored all my scripts in.
- I added the necessary codes to the CSS file too

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      blurEffect[i].classList.add('blurring');
+    }
+
+    isDisplay = false;
+    }
+    else {
+    hamburgerBtn.style.display = 'block';
+    times.style.display = 'none';
+    closeBar.classList.replace('icon-menu', 'nav-menu');
+    lists.classList.replace('js-nav-list', 'dv-nav');
```
Comment (brytebee) [inline main.js:26]: - [ ] **Required:** You have done all done an amazing job so far. 👏👏👏 However, when the hamburger is clicked and the menu is displayed, it isn't a good UI experience to have the page keep scrolling. Kindly use this [resource](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp) to hide the scrolling bar when the menu is displayed and show it when it isn't.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- good readme✔️
- all linters are installed and passed✔️
- mobile menu displayed and changed when clicked✔️
- the correct section is displayed when selected on the mobile✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (Ibnballo1) [reply main.js:26]: O thanks, i skipped that part as it is not even in the instruction given to us...
Will work on that now and I hope my project will e then approved

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Ibnballo1 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**